examples/user_groups/main.tf

```
@@ -0,0 +1,41 @@
+terraform{
+    required_providers {
+        nutanix = {
+            source = "nutanix/nutanix"
+            version = "1.6.2"
+        }
+    }
+}
+
+#definig nutanix configuration
+provider "nutanix"{
+  username = var.nutanix_username
+  password = var.nutanix_password
+  endpoint = var.nutanix_endpoint
+  port = 9440
+  insecure = true
+}
+
+# Add a User group to the system.
+
+# Add Directory Service user group.
+resource "nutanix_user_groups" "t1" {
+	directory_service_user_group{
+		distinguished_name = "{{distinguished name of the user group}}"
```

```suggestion
		distinguished_name = "<distinguished name of the user group>"
```
So that customer dont get confused if they have to add {{

---

Please make changes everywhere